src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -872,16 +1068,12 @@ public void testStartComposedArchiveWithLayout() throws OpenTokException {
                 .outputMode(OutputMode.COMPOSED)
                 .layout(new ArchiveLayout(ArchiveLayout.Type.CUSTOM, "stream { position: absolute; }"))
                 .build();
-
         Archive archive = sdk.startArchive(sessionId, properties);
         assertNotNull(archive);
         assertEquals(sessionId, archive.getSessionId());
         assertNotNull(archive.getId());
         assertEquals(OutputMode.COMPOSED, archive.getOutputMode());
-
         verify(postRequestedFor(urlMatching(archivePath)));
-                // TODO: find a way to match JSON without caring about spacing
```

Comment:
I don't think we should be removing these comments

Reply:
https://github.com/opentok/Opentok-Java-SDK/pull/150#discussion_r212050754